src/lib/profiles/data-management/Current/SubscriptionClient.cpp

```diff
@@ -2916,6 +2916,7 @@ void SubscriptionClient::AbortUpdates(WEAVE_ERROR aErr)
 
         numInProgress = mInProgressUpdateList.GetNumItems();
         mInProgressUpdateList.Clear();
+        SetPendingSetState(kPendingSetEmpty);
```

Comment:
Nit: add a newline above to emphasize this is a separate action from cleaning `mInProgressUpdateList`

Reply:
I would prefer this line to go next to the mPendingUpdateSet.Clear() line above, since the state of the two variables are tightly coupled.

Reply:
Done, thanks